src/__tests__/pages/api/soknad/faktum/save.test.ts

```diff
@@ -37,7 +37,7 @@ const faktumMockData: QuizFaktum = {
 const saveFaktumMockData: ISaveFaktumBody = {
   faktum: faktumMockData,
   svar: faktumMockData.gyldigeValg[0],
-  uuid: "1234",
+  uuid: "ce035d7f-c2a7-4954-a103-60402bdf69ac", // mock uuidv4
```

Comment:
Trenger man en kommentar som sier at det er mock-data i et mock-objekt? 🤔

Reply:
Nei, trenger egentlig ikke. Skulle bare teste en ting. Var bare note to self og glemte å fjerne :D .

Reply:
Men burde vi hatt en test på ugyldig uuid?

Reply:
@JMLindseth Done